index.html

```diff
@@ -254,8 +259,42 @@ <h3>LET'S CONNECT</h3>
                         
                     </ul>
                 </div>
+                <!-- start of contact form -->
+                <footer>
+                    <section class="contact-container">
+                        <div class="heading">
+                            <h2>Contact me</h2>
+                        </div>
+                        <div class="content">
+                            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+                        </div>
+                        <div class="form-container">
+                            <form
+                            action="https://formspree.io/f/mqkngnbr"
+                            method="POST"
+                          >
+                                <input type="text" id="name" name="username" required 
+                                maxlength="30" value="jemutai"><br><br>
+                          
+                                <input type="Email" id="email" name="email" required
+                                 value="kiplagatjemutai@gmail.com"><br><br>
```

Comment:
- [ ] I kindly suggest that you do not set the value of the input, but instead use a placeholder because the placeholder will give the user an example of the expected input value.

### EXAMPLE
`<input name="fullname" type="text" placeholder="John Doe" maxlength="30" aria-required="true" required="">`